### DISCUSSION TITLE
Failed to build my Xamarin Forms UWP project in release mode

### DISCUSSION THREAD

**Discussion by vsfeedback** [xamarin/Xamarin.Forms#11008] [General]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/content/problem/1068294/error-al-compilar-mi-proyecto-xamarin-forms-uwp-en.html)._

---

Right now I have a problem with a Xamarin Forms for UWP project, because when trying to build this in Mode
release, the compiler gets an error in which I don't know what it is and doesn't give me any more information about this.

" .nuget.packages-microsoft.net.native.compiler-2.1.8-tools-Microsoft.NetNative.targets(788,5): Error : Internal compiler error: Specified argument was out of the range of valid values. "

Thank you very much for the attention and if you can give me a hand to be able to fix it I would be hugely grateful

---
#### Visual Studio Feedback System on 6/8/2020, 00:05 AM: 

<p>We have directed your feedback to the appropriate engineering team for further evaluation. The team will review the feedback and notify you about the next steps.</p>

---
### Original Solutions
(no solutions)

**Comment by hartez** [comment]
Do you have "Compile with .NET Native tool chain" checked in your UWP project properties for Release mode? If so, does the build complete successfully if you uncheck this option?

![image](https://user-images.githubusercontent.com/538025/84318459-b3667080-ab2b-11ea-92c2-0ee31e435997.png)

**Comment by alejandroMcu** [comment]
![Captura](https://user-images.githubusercontent.com/3452202/84556274-194b2780-ace7-11ea-8739-6cd7e9e3429d.PNG)

**Comment by alejandroMcu** [comment]
hello hartez.

This is my project properties, have some bad to this error is present when i try to compile in release mode

**Comment by alejandroMcu** [comment]
This is the error when i try to compile

![Error](https://user-images.githubusercontent.com/3452202/84556726-2ddcef00-acea-11ea-915d-2de73653b4ea.PNG)

**Comment by MagicAndre1981** [comment]
Release mode doesn't accept AnyCPU for UWP, you need x86, x64, arm or arm64. For ARM64, also add [Win2D package to UWP app](https://www.nuget.org/packages/Win2D.uwp)

**Comment by alejandroMcu** [comment]
@MagicAndre1981 although i have configured muy project to deploy to x86, the error is present when i try of compile. My project is a Xamarin.Forms to UWP. Must configure the PCL project to run in release mode.

With small projects, only with a label that show a message, release mode run good, but when to run a project with graphical interface very much big, the error in present

**Comment by alejandroMcu** [comment]
Hello friends.

You have some solution for this issue. I have update Visual Studio to last version release but the error still continue. In small app when compile in release mode this don't happen, but, when the app grow, the issue is present.

Thank you for your help.
